.travis.yml

```diff
@@ -5,7 +5,7 @@ sudo: false
 install:
   - nvm install 6
   - nvm use 6
-  - npm install -g purescript@0.10.2 pulp bower
+  - npm install -g purescript@0.10.7 pulp@10.0.4 bower
```

Comment:
You can specify `0.11.1` and `11.0.0` here now. They are on NPM, but labelled `next`.